src/devices/digilux.js

```diff
@@ -0,0 +1,147 @@
+const modernExtend_1 = require("../lib/modernExtend");
```

Comment:
Please convert to `.ts`, example:

https://github.com/Koenkk/zigbee-herdsman-converters/blob/28685042a3d7e94fcacb850d5dd1e384b1969f21/src/devices/aeotec.ts#L31